Automattic/wp-calypso
# Site Assembler: Remove client-side-render feature flag

Related to #

## Proposed Changes

* Remove the `pattern-assembler/client-side-render` flag since it's been enabled on production for a long time, and the performance looks great now 🙂
* Use `React.lazy` to load the pattern assembler step lazily since it requires `@wordpress/edit-site` package and its size is a bit large

## Testing Instructions



* Go to `/setup?siteSlug=<your_site>`
* Continue until you land on the Design Picker
* On the Design Picker screen, scroll down to the bottom and select BCPA CTA
* On the Pattern Assembler screen, the functionalities should work well as before

## Pre-merge Checklist
